cmd/kind/get/get.go

```diff
@@ -32,11 +33,12 @@ func NewCommand() *cobra.Command {
 		// TODO(bentheelder): more detailed usage
 		Use:   "get",
 		Short: "Gets one of [clusters, nodes, kubeconfig-path]",
```

Comment:
this should also be in short. We should rewrite long at some point

Reply:
done